lib/Conversion/TosaToEmitC/TosaToEmitC.cpp

```diff
@@ -415,8 +442,9 @@ class RsqrtOpConversion : public OpConversionPattern<tosa::RsqrtOp> {
 };
 
 /// Convert `tosa.fully_connected` into an `emitc.call` operation.
```

Comment:
This comment seems off.

Reply:
Oops, I actually fixed it before but it got lost somewhere in the process. Fixed it again :)